spec/SPEC.md

```diff
@@ -100,21 +104,21 @@ time.
 
 ##	Scope
 
-Specifies procedures for constructing and conveying Artifact Dependency Graphs
-(ADGs), and other related data structures for artifacts. Including but not
-limited to:
+Specifies procedures for constructing and conveying Input Manifests,
+Artifact Dependency Graphs (ADGs), and other related data structures 
+for artifacts. Including but not limited to:
 
 - formats for artifact identifiers
 - formats for specifying graph relationships between artifacts
-- manner of embedding identifiers for ADGs, and other related
+- manner of embedding identifiers for Input Manifests, ADGs, and other related
   data structures in artifacts of various types
-- guidance on metadata which references ADGs, and other related data structures
+- guidance on metadata which references Input Manifests, ADGs, and other related data structures
 - guidance for build tools for:
- - constructing ADGs, and other related data structures
- - conveying ADGs, and other related data structures
- - embedding identifiers for ADGs, and other related data structures ids in artifacts
- - manners of conveyance of ADGs, and other related data structures
- - descriptions of use cases for which ADGs, and other related data structures may be used
+  - constructing Input Manifests, ADGs, and other related data structures
+  - conveying Input Manifests, ADGs, and other related data structures
+  - embedding identifiers for Input Manifests, and other related data structure's ids in artifacts
```

Comment:
"structure's" -> "structurs'" since these are plural possessives. Same on the next line.

Reply:
I think I fixed this in https://github.com/omnibor/spec/pull/51/commits/9e9cf01385226c565176b6a46802dfe8d0df9c46

But my recollection of how to do plural possessives would have "structure's" -> "structures'" so wanted to check in.